default/AI/ShipDesignAI.py

```diff
@@ -633,11 +635,11 @@ def evaluate(self):
         # However, we also need to make sure, that the closer we are to requirements,
         # the better our rating is so the optimizing heuristic finds "the right way".
         if self.fuel < self.additional_specifications.minimum_fuel:
-            rating += -1000 * (self.additional_specifications.minimum_fuel - self.fuel)
+            rating += MISSING_REQUIREMENT_MULTIPLIER * (self.additional_specifications.minimum_fuel - self.fuel)
```

Comment:
Looks like I missed it a little, but in more common cases I got right result.  I think we should pay attention for all new commits for this case and leave existing code untouched.

PEP8
> Use your own judgment; however, never use more than one space, and always have the same amount of whitespace on both sides of a binary operator